consensus/safety-rules/src/safety_rules.rs

```diff
@@ -230,11 +243,44 @@ impl<T: Payload> TSafetyRules<T> for SafetyRules<T> {
         ))
     }
 
-    /// @TODO only sign blocks that are later than last_voted_round and match the current epoch
-    /// @TODO verify QC correctness
-    /// @TODO verify QC matches preferred round
+    /// @TODO consider the API change: BlockData<T> to Block<T>
```

Comment:
let's remove this, I don't think this is possible, the Block<T> is the return value which contains the signature

Reply:
hahaha, did I mix two apis up! I intended this to be for sign and construct vote :P

@zekun000 is right!